### ISSUE TITLE
[UX sort of bug] The auto-completion started by a colon is rather confusing

### ISSUE THREAD

**Issue by ell1e** [vector-im/element-web#23088]
#### What would you like to do?
The auto-completion started by a colon is rather confusing and unlike any other messengers I use. The *expected* behavior is that if I type `:jo` and press tab immediately, I get `:joy:`.

This is what actually happens:

1. If I do this too fast, nothing. The completion popup shows up late but my tab press that already happened is ignored. This is very frustrating

2. If there is a user around, even though users are *even shown with `@` and not `:`* (and in all other messengers I use, you need an `@` to start a user auto completion), and the user has a name that starts similarly, I might suddenly auto-complete to the user.

3. Only if none of the above two fail, I get what I actually intended.

This makes feel element feel unreliable and moody, and most importantly, it messes up muscle memory. The emoji auto completion shouldn't be timing dependent and *predictable*, these seem to be broken by the strange throwing in of users and the delayed popping up - which I'm suspecting might also be due to the users being loaded or queried from the server?

TL;DR: please make the emoji auto complete not fail if I type too fast and not auto-complete the users if I typed `:` and not `@` (so I clearly want an emoji and not a user, one would think?)

#### Why would you like to do it?
The changes suggested above would be nice so using emojis in element isn't so frustrating and weird

#### How would you like to achieve it?
See above

### Have you considered any alternatives?

The other messengers work fine with this. Also, I guess I could type really slowly, but I'm hoping that nobody is seriously suggesting this as a good soluton

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/9757
